uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Page/AbsenceType.php

```diff
@@ -21,7 +21,7 @@ public function run() {
 
   public function browse() {
     $object = new CRM_HRLeaveAndAbsences_BAO_AbsenceType();
-    $object->orderBy('category');
+    $object->orderBy('weight');
```

Comment:
The PR description says that the absence type is ordered by category, but it is ordered by weight here?

Reply:
There was a misinterpretation of one of the ordering requirements, calling for this change. I will update the description.